Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderContent = () => {
+    return (
+        <div className="header__content"><p>
```

Comment:
This `<p>` is redundant because you can just have text directly in the parent div if you aren't styling specifically for `<p>` tags on your pages.